scripts/anaconda-pre-log-gen.sh

```diff
@@ -0,0 +1,12 @@
+#! /bin/bash
+# Script to be run by anaconda-pre service before anaconda starts to gather
+# information about previous system state
+
+TARGET_DIRECTORY=/tmp/pre-anaconda-logs
+
+mkdir ${TARGET_DIRECTORY}
+
+lsblk -a >${TARGET_DIRECTORY}/block_devices.log
+dmesg -a >${TARGET_DIRECTORY}/block_devices.log
+
+lvmdump
```

Comment:
Use option -d with the directory. With this setup you will create ``lvmdump`` to /root.